arangod/Replication/Syncer.cpp

```diff
@@ -910,7 +910,7 @@ void Syncer::reloadUsers() {
   AuthenticationFeature* af = AuthenticationFeature::instance();
   auth::UserManager* um = af->userManager();
   if (um != nullptr) {
-    um->outdate();
+    um->setGlobalVersion(um->globalVersion()+1);
```

Comment:
Is this thread-safe?
There may be concurrent callers to `setGlobalVersion` from other threads, and in this case we may lose an update.

I think we should rather add `UserManager::increaseGlobalVersion()` to atomically increment the global version.